### PR TITLE
chore: remove inactive reviewers

### PR DESCRIPTION
**Pull request by Bobgy**
**Description of your changes:**
Remove inactive reviewers from OWNERS, so that external PRs are assigned to active maintainers.

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

   PR titles examples:
    * `fix(frontend): fixes empty page. Fixes #1234`
       Use `fix` to indicate that this PR fixes a bug.
    * `feat(backend): configurable service account. Fixes #1234, fixes #1235`
       Use `feat` to indicate that this PR adds a new feature. 
    * `chore: set up changelog generation tools`
       Use `chore` to indicate that this PR makes some changes that users don't need to know.
    * `test: fix CI failure. Part of #1234`
        Use `part of` to indicate that a PR is working on an issue, but shouldn't close the issue when merged.

### CODE REVIEW

**Comment by Bobgy** [comment]
Any concerns?

File: backend/OWNERS [hunk 1]
```diff
@@ -4,7 +4,5 @@ approvers:
   - neuromage
   - rmgogogo
 reviewers:
-  - IronPan
+  - Bobgy
   - jingzhang36
-  - neuromage
-  - rmgogogo
```
Comment (rmgogogo) [inline backend/OWNERS:10]: please keep me in the list

Comment (Bobgy) [reply backend/OWNERS:10]: sure, will do

**Pull request merged**